### ISSUE TITLE
Custom DoH

### ISSUE THREAD

**Issue by LittleRey** [tachiyomiorg/tachiyomi#6426]
### Describe your suggested feature

thanks

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have updated the app to version **[0.12.3](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by alessandrojean** [comment]
Can't you use a custom DoH system-wide?

**Comment by LittleRey** [comment]
@alessandrojean I means this software listed DOH Too little，Please allow users to customize DOH service

**Comment by loocool2** [comment]
I don't think most users would add to the list as many don't even know what DNS over HTTPS or where to get more DoH options. Is the current list not sufficient for your uses? Are the three currently in the app all blocked by your country or some such?

**Comment by LittleRey** [comment]
@loocool2 yes,you are right,cloudflare, Google,Adguard DOH is blocked in my country

**Comment by alessandrojean** [comment]
This is a very specific use case and it would still be better if you just use the DoH system-wide in the android connectivity settings, so you can change it any time and it would apply to all apps.

**Comment by LittleRey** [comment]
@alessandrojean I know u talk about,u means let me use Android Private DNS,It's Dot, the all of 853 port is blocked in my country,so this service no work in my country,:sob::sob:

**Comment by arkon** [comment]
You're better off using actual VPNs, DNS over HTTPS isn't going to be that useful for your needs in what I assume is China.
